addons/binding/org.openhab.binding.nest/ESH-INF/thing/camera.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="camera" listed="false">
```

Comment:
Mainly for turning the camera on and off.  In theory we can get events for the camera too.

Yes, turning it on and off is useful though.  The use case for this is the home occupied state.  If the home is occupied then turn it on, if it is not occupied then turn it off.

Nest is working on a push channel too which will allow us to get events and we can then push these into openhab as well.